data/import.graphql

```diff
@@ -1,5 +1,6 @@
 type ImportStatus{
     error: String
+    warnings: [String!]!
```

Comment:
Can we make the output nullable so that it's consistent with error and more "standard-ish"?